test/simulation_and_solving/simulate_jumps.jl

```diff
@@ -16,12 +16,17 @@ include("../test_networks.jl")
 
 # Compares jump simulations generated through hCatalyst, and manually created systems.
 let
-    # Manually declares jumps to compare Catalyst-generated jump simulations to.
+    # Declares vectors to store information about each network in. Initial conditions are approximately
```

Comment:
I have added manually checked parameter sets and initial conditions, and for each checked that the mean are reproducibly similar across very many simualtions.